docs/source/examples/spot-jobs.rst

```diff
@@ -313,7 +313,12 @@ Spot Pipeline
 
 Spot Pipeline is a feature that allows you to submit a spot job that contains a sequence of spot tasks running one after another.
 This is useful for running a sequence of jobs that depend on each other, e.g., training a model and then running inference on it.
-This allows the multiple tasks to have different resource requirements to fully utilize the resources and save cost, while keep the burden of managing the tasks off the user. 
+This allows the multiple tasks to have different resource requirements to fully utilize the resources and save cost, while keeping the burden of managing the tasks off the user. 
+
+.. note::
+  A job is submitted by :code:`sky spot launch`, it can be a single task or a pipeline of tasks.
+  
+  All tasks in a pipeline will be run on spot instances. SkyPilot do not support using on-demand instances for pipelines yet.
```

Comment:
```suggestion
  A job is either a single task or a pipeline of tasks, submitted by :code:`sky spot launch`.
  
  All tasks in a pipeline will be run on spot instances.
```
(Removing the on-demand note as Kapa may misunderstand it.)

Reply:
Thanks for the change! I changed `job` to `spot job` to avoid the confusion with a normal job. Also, I explicitly say that `a spot job is submitted by sky spot launch`, as the original one can be a bit confusing whether it is the task or the job that is submitted by `sky spot launch`.